src/modules/addRemove.js

```diff
@@ -0,0 +1,28 @@
+class Task {
+  constructor(description, index = 0, isComplete = false) {
+    this.description = description;
+    this.isComplete = isComplete;
+    this.index = index;
+  }
+
+  addTask = (task, tasks) => {
+    tasks.push(task);
+    localStorage.setItem('tasks', JSON.stringify(tasks));
+  }
```

Comment:
<img width="398" alt="image" src="https://user-images.githubusercontent.com/50707800/184672951-689eb85b-de85-442f-b185-84e3e066bab9.png">
make change your empty submitting task.

Reply:
This was also something I did not notice. Consider it worked on.